staging/src/k8s.io/client-go/pkg/kubelet/qos/doc.go

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
We are trying to remove the packages in /pkg/kubelet,  which are are duplicate packages between kuberentes and client-go. Are there other users of `pkg/kubelet/qos` except for client-go? If so, we can move this package to client-go, and remove the original copy. cc @deads2k @sttts